src/terra-dev-site/example/ResponsiveListAndCellGridExample.jsx

```diff
@@ -0,0 +1,174 @@
+
+const display1 = <ItemView.Display icon={<IconPerson />} text="Asif Khan" />;
+const display2 = <ItemView.Display text="Care Position: Primary" />;
+const display3 = <ItemView.Display text="Room 100A" />;
+const display4 = <ItemView.Display text="Acuity: 5" />;
+const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
+const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
+const comment = <ItemView.Comment text="Faint red rash appeared at 08-05-2016 13:24:00" />;
+const startAccessory = <IconAlert />;
+const endAccessory = <IconInformation />;
+
+const cx = classNames.bind(styles);
+
+const listDisplay = (
+  <List role="listbox">
+    <Item key="default">
+      <ItemView
+        displays={[
+          <ItemView.Display icon={<IconPerson />} text="Asif Khan" textStyle="attention" />,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+        endAccessory={endAccessory}
+      />
+    </Item>
+    <Item key="default">
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          <ItemView.Display text="Acuity: 5" icon={<IconAttachment />} />,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+        endAccessory={endAccessory}
+        layout="twoColumns"
+      />
+    </Item>
+    <Item key="default">
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        reserveStartAccessorySpace
+        comment={comment}
+        endAccessory={endAccessory}
+      />
+    </Item>
+    <Item key="default">
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          <ItemView.Display text="Room 100A" textStyle="strikeThrough" />,
+          display4,
+          display5,
+          display6,
+        ]}
+        reserveStartAccessorySpace
+        comment={comment}
+        endAccessory={endAccessory}
+        layout="twoColumns"
+      />
+    </Item>
+    <Item key="default">
+      <ItemView
+        displays={[
+          display1,
+          display2,
+          display3,
+          display4,
+          display5,
+          display6,
+        ]}
+        startAccessory={startAccessory}
+        comment={comment}
+      />
+    </Item>
+  </List>
+);
+
+const cellGridDisplay = (
+  <div className={cx('cellgrid-wrapper')}>
+    <CellGrid className={cx('row')}>
+      <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }}>{startAccessory}</Cell>
+      <Cell className={cx('display')} key="cell-1" width={{ scalar: 1 }}><ItemView.Display icon={<IconPerson />} text="Asif Khan" textStyle="attention" /></Cell>
```

Comment:
we could also define these item views once and reuse them in both list and cellgrid displays, like you do with the others